pkg/BUILD

```diff
@@ -0,0 +1,117 @@
+load("@rules_pkg//:pkg.bzl", "pkg_zip")
```

Comment:
I don't think there is an easy way to do that diff... the new file is largely cut-and-pasted, with a bit of reorganizing and updating deps to be absolute.

One thing that might be notable is that I expanded `//:wkt_proto_types` directly, and split out the descriptor sources at the same time (since it's not technically part of the Well Known Types).

The major addition are the two cc_dist_library at the bottom.